test/integration/model/geography.test.js

```diff
@@ -0,0 +1,189 @@
+          return Pub.create({location: point});
+        }).then(function (pub) {
+          expect(pub).not.to.be.null;
+          expect(pub.location).to.be.deep.eql(point);
+        });
+      });
+
+      it('should create a geography object', function() {
+        var User = this.User;
+        var point = { type: 'Point', coordinates: [39.807222,-76.984722]};
+
+        return User.create({username: 'username', geography: point }).then(function(newUser) {
+          expect(newUser).not.to.be.null;
+          expect(newUser.geography).to.be.deep.eql(point);
+        });
+      });
+
+      it('should update a geography object', function() {
+        var User = this.User;
+        var point1 = { type: 'Point', coordinates: [39.807222,-76.984722]}
+          , point2 = { type: 'Point', coordinates: [49.807222,-86.984722]};
+        var props = {username: 'username', geography: point1};
+
+        return User.create(props).then(function(user) {
+          return User.update({geography: point2}, {where: {username: props.username}});
+        }).then(function(count) {
+          return User.findOne({where: {username: props.username}});
+        }).then(function(user) {
+          expect(user.geography).to.be.deep.eql(point2);
+        });
+      });
+    });
+
+    describe('GEOGRAPHY(POINT)', function() {
+      beforeEach(function() {
+        return Support.prepareTransactionTest(this.sequelize).bind(this).then(function(sequelize) {
+          this.User = this.sequelize.define('User', {
+            username: DataTypes.STRING,
+            geography: DataTypes.GEOGRAPHY('POINT')
+          });
+
+          return this.User.sync({ force: true });
+        });
+      });
+
+      it('should create a geography object', function() {
+        var User = this.User;
+        var point = { type: 'Point', coordinates: [39.807222,-76.984722]};
+
+        return User.create({username: 'username', geography: point }).then(function(newUser) {
+          expect(newUser).not.to.be.null;
+          expect(newUser.geography).to.be.deep.eql(point);
+        });
+      });
+
+      it('should update a geography object', function() {
+        var User = this.User;
+        var point1 = { type: 'Point', coordinates: [39.807222,-76.984722]}
+          , point2 = { type: 'Point', coordinates: [49.807222,-86.984722]};
+        var props = {username: 'username', geography: point1};
+
+        return User.create(props).then(function(user) {
+          return User.update({geography: point2}, {where: {username: props.username}});
+        }).then(function(count) {
+          return User.findOne({where: {username: props.username}});
+        }).then(function(user) {
+          expect(user.geography).to.be.deep.eql(point2);
+        });
+      });
+    });
+
+    describe('GEOGRAPHY(LINESTRING)', function() {
+      beforeEach(function() {
+        return Support.prepareTransactionTest(this.sequelize).bind(this).then(function(sequelize) {
```

Comment:
I think you should be able to remove the `prepareTransactionTest` part - I know the other geom tests use it, but I dunno why :)